token/services/ttxdb/driver/driver.go

```diff
@@ -99,39 +99,73 @@ type TransactionIterator interface {
 	Next() (*TransactionRecord, error)
 }
 
-// AuditDB defines the interface for an audit database
-type AuditDB interface {
-	// Close closes the audit database
+// QueryMovementsParams defines the parameters for querying movements
```

Comment:
It is not clear how the movement is defined in relation to the enrollment IDs. It would be good to clarify this.